docs/getting-started.md

```diff
@@ -24,32 +24,30 @@ npx react-native init <projectName> --template react-native@^0.63.2
 
 Once your project has been initialized, React Native will have created a new sub directory where all your generated files live.
 
-```
-cd <projectName>
+```bat
+cd projectName
 ```
 
 ### Install the Windows extension
 
 Lastly, install the React Native for Windows packages.
 
-```
+```bat
 npx react-native-windows-init --overwrite
 ```
 
 > The --overwrite flag is a temporary measure that ensures the correct files are copied to metro.config.js for the metro bundler to work with Windows. If you are starting a new app, this should have no impact. If you are adding Windows to your existing app and you have modified the metro.config.js file, please back up your changes, run the command and copy over to take effect. We are tracking [this issue here](https://github.com/microsoft/react-native-windows/issues/4698).
 
 Here are the options that `react-native-windows-init` takes:
-```none
-Options:
-  --help       Show help                                               [boolean]
-  --version    The version of react-native-windows to use.              [string]
-  --namespace  The native project namespace.                            [string]
-  --verbose    Enables logging.                                        [boolean]
-  --language   Which language the app is written in.
-                                [string] [choices: "cs", "cpp"] [default: "cpp"]
-  --overwrite  Overwrite any existing files without prompting          [boolean]
-  --useWinUI3  Targets WinUI 3.0 (Preview) instead of UWP XAML         [boolean]
-  ```
+| Option      | Input Type                           | Description                                      |
+|-------------|--------------------------------------|--------------------------------------------------|
+| --help      | boolean                              | Show help.                                       |
+| --version   | string                               | The version of react-native-windows to use.      |
+| --namespace | string                               | The native project namespace.                    | 
+| --verbose   | boolean                              | Enables logging.                                 |
+| --language  | string ["cs","cpp"] [default: "cpp"] | Which language the app is written in.            |
+| --overwrite | boolean                              | Overwrite any existing files without prompting.  |
+| --useWinUI3 | boolean                              | Targets WinUI 3.0 (Preview) instead of UWP XAML. |

```

Comment:
```suggestion
| Option      | Input Type                           | Description                                      |
|-------------|--------------------------------------|--------------------------------------------------|
| `--help`      | boolean                              | Show help.                                       |
| `--version`  | string                               | The version of react-native-windows to use.      |
| `--namespace` | string                               | The native project namespace.                    | 
| `--verbose`   | boolean                              | Enables logging.                                 |
| `--language`  | string ["cs","cpp"] [default: "cpp"] | Which language the app is written in.            |
| `--overwrite` | boolean                              | Overwrite any existing files without prompting.  |
| `--useWinUI3` | boolean                              | Targets WinUI 3.0 (Preview) instead of UWP XAML. |
```